<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MindMed</class>
 <widget class="QMainWindow" name="MindMed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MindMed</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(98, 132, 255);
border-color: rgb(85, 85, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>login</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>225</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Administrator Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b01">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>290</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(71, 73, 194);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tb02">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>User Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="l01">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Enter user name and password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tb01">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>210</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mental Health Management System</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>home</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>450</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Welcome to our mental health management system</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(85, 85, 255);</string>
   </property>
   <widget class="QMenu" name="menuemployee">
    <property name="title">
     <string>Employee</string>
    </property>
    <addaction name="menu11"/>
    <addaction name="menu12"/>
    <addaction name="menu13"/>
    <addaction name="menu14"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuMember">
    <property name="title">
     <string>Member</string>
    </property>
    <addaction name="menu21"/>
    <addaction name="menu22"/>
    <addaction name="menu23"/>
    <addaction name="menu24"/>
   </widget>
   <widget class="QMenu" name="menuPomodoro">
    <property name="title">
     <string>Pomodoro</string>
    </property>
    <addaction name="menu31"/>
   </widget>
   <widget class="QMenu" name="menuMindfulness">
    <property name="title">
     <string>Mindfulness</string>
    </property>
    <addaction name="menu41"/>
    <addaction name="menu42"/>
   </widget>
   <widget class="QMenu" name="menuMod_Tracker">
    <property name="title">
     <string>Mod Tracker</string>
    </property>
    <addaction name="menu51"/>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="title">
     <string>Report</string>
    </property>
    <addaction name="menu61"/>
    <addaction name="menu62"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="menu7"/>
   </widget>
   <addaction name="menuemployee"/>
   <addaction name="menuMember"/>
   <addaction name="menuPomodoro"/>
   <addaction name="menuMindfulness"/>
   <addaction name="menuMod_Tracker"/>
   <addaction name="menuReport"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu11">
   <property name="text">
    <string>Add New Employee</string>
   </property>
  </action>
  <action name="menu12">
   <property name="text">
    <string>Edit Employee</string>
   </property>
  </action>
  <action name="menu13">
   <property name="text">
    <string>Delete Employee</string>
   </property>
  </action>
  <action name="menu21">
   <property name="text">
    <string>Add New Member</string>
   </property>
  </action>
  <action name="menu22">
   <property name="text">
    <string>Edit Member</string>
   </property>
  </action>
  <action name="menu23">
   <property name="text">
    <string>Delete Member</string>
   </property>
  </action>
  <action name="menu24">
   <property name="text">
    <string>Search Member</string>
   </property>
  </action>
  <action name="menu14">
   <property name="text">
    <string>Search Employee</string>
   </property>
  </action>
  <action name="menu31">
   <property name="text">
    <string>Start session</string>
   </property>
  </action>
  <action name="menu41">
   <property name="text">
    <string>About MF</string>
   </property>
  </action>
  <action name="menu42">
   <property name="text">
    <string>Start session</string>
   </property>
  </action>
  <action name="menu51">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="menu61">
   <property name="text">
    <string>Daily Activties Report</string>
   </property>
  </action>
  <action name="menu62">
   <property name="text">
    <string>Indvual Member Report</string>
   </property>
  </action>
  <action name="menu7">
   <property name="text">
    <string>About MindMed</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
